v2/loader/schema_parser_source.go

```diff
@@ -21,13 +21,21 @@ package loader
 
 import (
 	"fmt"
+	"github.com/cloudspannerecosystem/memefish/ast"
```

Comment:
I would be pleased if the standard packages and others are grouped separately.

Reply:
Done https://github.com/cloudspannerecosystem/yo/pull/145/commits/6d6da2649737fc19dfdae12b359cd4b3dd27125b